prebid-mobile/pbm-api/ios/pbm-targeting-ios.md

```diff
@@ -48,6 +48,7 @@ Prebid.shared.customStatusEndpoint="https://pbs.example.com/v2/status"
 | prebidServerAccountId | either | string | init | Your Prebid Server team will tell you whether this is required or not and if so, the value. | "abc123" |
 | prebidServerHost | optional | enum | init | This can take the values "Appnexus", "Rubicon", or "Custom". If "Custom", you need to use the setCustomPrebidServerUrl() method to set a URL. This is where the Prebid SDK will send the auction information. Your Prebid Server team will tell you which value to use. The default is "Custom". | "Custom" |
 | customStatusEndpoint | optional | string | init | Use this URL to check the status of Prebid Server. The default status endpoint is the PBS URL appended with '/status'. | `https://prebidserver`<wbr>`.example`<wbr>`.com/custom`<wbr>`/status` |
+| auctionSettingsId | optional | string | init | For the SDK to separate out account from "auction settings". If specified, the auctionSettingsId is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is populated by default. | "abc321" |
```

Comment:
```suggestion
| auctionSettingsId | optional | string | init | For the SDK to separate account from "auction settings", allowing each app to have different global parameters defined on the server side. If specified, the auctionSettingsId is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is populated by default. | "abc321" |
```